### DISCUSSION TITLE
Feature Request: Esplora/Mempool.space testnet

### DISCUSSION THREAD

**Discussion by realeinherjar** [raspiblitz/raspiblitz#4070] [Research]
Is it possible to run mempool in parallel with mainnet/testnet/signet?
I want not only to use the mempool.space explorer but also the esplora API.
<img width="382" alt="image" src="https://github.com/raspiblitz/raspiblitz/assets/136860407/6693e129-468e-410f-9752-e273f1b4ecbf">

**Comment by rootzoll** [comment]
It is possible to run mainnet, testnet & signet all at the same time on RaspiBlitz. How to confige that with mempool would be a community reasearch topic - so I moved this issue to RESEARCH discussions.
